stripe/src/main/java/com/stripe/android/view/CardMultilineWidget.kt

```diff
@@ -687,19 +686,20 @@ class CardMultilineWidget @JvmOverloads constructor(
     }
 
     private fun initFocusChangeListeners() {
-        cardNumberEditText.onFocusChangeListener = OnFocusChangeListener { _, hasFocus ->
+        cardNumberEditText.internalFocusChangeListener.add { _, hasFocus ->
             if (hasFocus) {
                 cardInputListener?.onFocusChange(CardInputListener.FocusField.CardNumber)
             }
         }
 
-        expiryDateEditText.onFocusChangeListener = OnFocusChangeListener { _, hasFocus ->
+        expiryDateEditText.internalFocusChangeListener.add { _, hasFocus ->
+            // TODO: Why is this only called when have focus?
```

Comment:
I intend to remove this comment, but I am curious if you know?

Reply:
`onFocusChange()` is called with the field that now has focus

Reply:
Would it make sense to name it onFocusGained?  Not considering it for this PR, but might be worth thinking about if we are in this code again.